File: 1 лаба/C++/G.cpp
#include <iostream>
#include <vector>
using namespace std;

void CountSort(vector<int> &arr) {
  int count[101] = {0};

  for (int i = 0; i < (int)arr.size(); i++) {
    count[arr[i]]++;
  }

  int idx = 0;
  for (int i = 0; i <= 100; i++) {
    while (count[i] > 0) {
      arr[idx] = i;
      idx++;
      count[i]--;
    }
  }
}

int main() {
  vector<int> arr;
  int x;
  while (cin >> x) {
    arr.push_back(x);
  }

  CountSort(arr);

  for (int i = 0; i < (int)arr.size(); i++) {
    cout << arr[i];
    if (i + 1 < (int)arr.size()) {
      cout << " ";
    }
  }
  cout << endl;

  return 0;
}

File: 1 лаба/Python/G.py
def CountSort(arr):
    count = [0] * 101
    for x in arr:
        count[x] += 1

    res = []
    for i in range(101):
        while count[i] > 0:
            res.append(i)
            count[i] -= 1
    return res


arr = list(map(int, input().split()))
print(*CountSort(arr))
